domains/client/domain-operator/src/tests.rs

```diff
@@ -998,6 +999,120 @@ async fn test_invalid_total_rewards_proof_creation() {
     ferdie.produce_blocks(1).await.unwrap();
 }
 
+#[tokio::test(flavor = "multi_thread")]
+#[ignore]
```

Comment:
The underlying test infra for Fraud proof is still in V1 and not fully implemented. We have another issue to handle the necessary changes but in the meantime I wanted to add the tests for the changes this PR brings, but skipped, so that the other issue can simply handle enabling the tests rather adding them.

Reply:
> The underlying test infra for Fraud proof is still in V1 and not fully implemented.

Do you mean https://github.com/subspace/subspace/issues/1527?